README.md

```diff
@@ -47,6 +47,10 @@ First Issue is a static website that uses Next.js, React and Typescript. The dat
 To contribute new features and changes to the website, you would want to run the app locally. Please follow these steps:
 
 ## How to setup the project locally
+To set up the project locally or to run it on your server, follow these steps:
+Run `yarn`.
```

Comment:
We are using `npm` ;)

Reply:
Thank you for your review, @lucavallin. I appreciate your feedback. Initially, I encountered some difficulties while trying to run this repository, but I've managed to resolve those issues, and I'll be working on the edits now. Your input is valuable, and I'm here to address any further concerns or suggestions you might have. Thanks again